spec/factories/offender_categories.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :offender_category, class: HmppsApi::OffenderCategory do
+    initialize_with { HmppsApi::OffenderCategory.new(attributes.stringify_keys) }
+
+    offenderNo { '' }
```

Comment:
not sure about defaulting this to blank - maybe better to force callers to fill it in? Feels like a mandatory has_one relationship

Reply:
Good point. I've refactored this factory to remove the need for `offenderNo` to be explicitly specified.